### ISSUE TITLE
Replace frames

### ISSUE THREAD

**Issue by neumartin** [iizukanao/node-rtsp-rtmp-server#135]
It's possible to replace frames, for example to draw somthing over.

**Comment by iizukanao** [comment]
No, this server doesn't have such feature. You should draw something before raw video frames are encoded into H.264.
